src/ipc/ipc-zephyr.c

```diff
@@ -254,7 +313,7 @@ enum task_state ipc_platform_do_cmd(struct ipc *ipc)
 void ipc_platform_complete_cmd(struct ipc *ipc)
 {
 	ARG_UNUSED(ipc);
-	intel_adsp_ipc_complete(INTEL_ADSP_IPC_HOST_DEV);
```

Comment:
does this break the "old" API? Shell it then be removed completely or do we need `#ifdef`s in these calls too?